src/styles/main.scss

```diff
@@ -0,0 +1,57 @@
+.clock {
+  height: 340px;
```

Comment:
This is optional task but when you get time try to work with variables for this task and try to formulate some values. 
Avoid hard code wherever you can